lib/components/sprite_stack_component.dart

```diff
@@ -0,0 +1,117 @@
+import 'dart:ui';
+
+import '../sprite.dart';
+import '../spritesheet.dart';
+import 'component.dart';
+
+class SpriteStackComponent extends PositionComponent {
+  SpriteSheet spriteSheet;
+
+  static final _sprites = <String, Map<double, Sprite>>{};
+
+  @override
+  set angle(double value) {
+    if (_sprites[spriteSheet.imageName][value] == null) {
+      _generated = false;
+    }
+    super.angle = value;
+  }
+
+  SpriteStackComponent(
+      {String imageName,
+      int textureWidth,
+      int textureHeight,
+      int columns,
+      int rows}) {
+    if (_sprites[imageName] == null) {
+      _sprites[imageName] = Map();
+    }
+    x = 100;
+    y = 100;
```

Comment:
why is this a default value?

Reply:
These were just some test values to see how it responded. In my current (unstable) version they have all been removed